.github/issue_template.md

```diff
@@ -1,18 +1,18 @@
-**Description:**

```

Comment:
Any reason to move issue template inside `.github` directory?

Reply:
Thought it will be much cleaner, with multiple files at the root level, it seemed bit cluttered

Reply:
Isn't that affects when we create issues?

Reply:
Yes the config looks for these under `.github` too. Check out https://github.com/ballerina-platform/module-ballerinax-mongodb/issues